### PR TITLE
Migration common toolkit

### PR DESCRIPTION
**Pull request by gmalinve**

### CODE REVIEW

**Comment by SMoraisAnsys** [comment]
We'll probably have to wait for a new release to be created to be sure that the tests are passing correctly.
The PR that should fix the issue is https://github.com/ansys/pyaedt/pull/4603

File: pyproject.toml [hunk 1]
```diff
@@ -43,7 +34,7 @@ tests = [
     "flask",
 ]
 doc = [
-    "pyaedt[all]==0.8.1",
+    "pyaedt[all]==0.8.9",
```
Comment (SMoraisAnsys) [inline pyproject.toml:37]:
```suggestion
    "pyaedt[all]==0.9.0",
```

File: pyproject.toml [hunk 2]
```diff
@@ -43,7 +34,7 @@ tests = [
     "flask",
 ]
 doc = [
-    "pyaedt[all]==0.8.1",
+    "pyaedt[all]==0.9.0",
```
Comment (SMoraisAnsys) [inline pyproject.toml:37]: Not sure if we couldn't just use "ansys-aedt-toolkits-common[doc]" here

**Review by SMoraisAnsys — DISMISSED** [review]

**Comment by SMoraisAnsys** [comment]
@gmalinve Seems like there is still a problem as we are facing the following issue

image

I won't have time to look into it before going into days off + ooo but feel free to contact me if I can help (or have a look)

**Comment by SMoraisAnsys** [comment]
@gmalinve I took the liberty of updating the README to fix the build package issue

**Review by SMoraisAnsys — DISMISSED** [review]

**Review by SMoraisAnsys — APPROVED** [review]

**Review by Samuelopez-ansys — APPROVED** [review]
Good job! Next week I will test the new release with the toolkit manager.

**Pull request merged**